### PR TITLE
Feature 4 create patients call

### PR DESCRIPTION
**Pull request by Thealean28**
## Description
The objective of this feature is to implement the creation of the patients that will be used this week in the development of the application.

## Related Issue(s)
- #4 

## Checklist
- [x] I have read and followed the project's contributing guidelines.
- [x] I have tested the changes locally.
- [x] I have made corresponding changes to the documentation if necessary.
- [x] My code follows the project's code style guidelines.

## Reviewers
- @alvarobernal2412 
- @gonzcm  

## Notes (if applicable)

### CODE REVIEW

File: src/core/database/session.py [hunk 1]
```diff
@@ -2,20 +2,26 @@
...
 from sqlalchemy.ext.asyncio import create_async_engine
 from sqlmodel.ext.asyncio.session import AsyncSession
+import asyncio
+import sys
 
 from src.core.config import settings
 
 DB_POOL_SIZE = 83
-WEB_CONCURRENCY = 9
+WEB_CONCURRENCY = 100
```
[the remark this answers is not in the document]
Comment (Thealean28) [reply src/core/database/session.py]: Done!

File: tests/api/test_acat_patient.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import pytest
+from src.modules.acat.patient.model import Patient
```
Comment (jmartinacu) [inline tests/api/test_acat_patient.py:2]: This import is not used in the file.

**Review by jmartinacu — CHANGES_REQUESTED** [review]
The code is not following all the rules specified by the code style PEP8

File: tests/api/test_acat_patient.py (hunk 2 above)
Comment (Thealean28) [reply tests/api/test_acat_patient.py]: Done!

Comment (gonzcm) [reply tests/api/test_acat_patient.py]: Well done!

File: src/core/database/session.py [hunk 3]
```diff
@@ -2,20 +2,26 @@
...
+    asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
+
 connect_args = {"check_same_thread": False}
 
 engine = create_async_engine(
     str(settings.SQLALCHEMY_DATABASE_URI),
     echo=False,
     future=True,
-    # pool_size=POOL_SIZE,
+    pool_size=POOL_SIZE,
```
[the remark this answers is not in the document]
Comment (gonzcm) [reply src/core/database/session.py]: Well done!

File: src/core/database/session.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (gonzcm) [reply src/core/database/session.py]: Well done!

File: src/modules/acat/patient/router.py [hunk 4]
```diff
@@ -1,8 +1,17 @@
...
 
 router = APIRouter()
 
 
 @router.get('/')
 def root():
     return 'Hello acat patient router!'
+
+@router.post('/')
+async def create_patient(session: SessionDep, patient: Patient) -> Any:
```
Comment (gonzcm) [inline src/modules/acat/patient/router.py:15]: Please add the return type of function.

**Review by gonzcm — CHANGES_REQUESTED** [review]

File: src/modules/acat/patient/router.py (hunk 4 above)
Comment (Thealean28) [reply src/modules/acat/patient/router.py]: Done!

**Review by alvarobernal2412 — APPROVED** [review]
Thank you for resolving conflicts! This task is necessary for the correct functioning of the app.

**Review by gonzcm — APPROVED** [review]
Good job!

**Pull request merged**